### ISSUE TITLE
Accessibility - Rewrite forms in search interface content type

### ISSUE THREAD

**Issue by axamei** [pulibrary/special_collections#406]
Producing accessibility errors. Missing appropriate labels.

**Comment by sandbergja** [comment]
@kevinreiss does this ticket refer to these webforms? https://library.princeton.edu/special-collections/admin/content/webform

If so, they appear to have labels now, and I think we can close this ticket.
